v1.1/admin_ui_cluster_overview.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Cluster Overview 
+toc: false
+feedback: false
+---
+
+CockroachDB’s Admin UI enables you to monitor metrics such as SQL Queries, Service Latency, Replicas per Node, and Capacity of your cluster.
+
+<div id="toc"></div>
+
+On [accessing the Admin UI](explore-the-admin-ui.html#access-the-admin-ui), the first dashboard you see by default is the Cluster Overview. 
+
+#### Cluster and Node View for the Time Series Graphs
+By default, the Time Series panel displays the cluster view, which shows the metrics for the entire cluster. 
+
+To access the node view that shows the metrics for an individual node, select the node from the **Graph** drop-down list.
+
+<img src="{{ 'images/admin_ui_select_node.png' | relative_url }}" alt="CockroachDB Admin UI" style="border:1px solid #eee;max-width:40%"/>
+
+The Cluster Overview displays the following time series graphs:
+
+## SQL Queries
+<img src="{{ 'images/admin_ui_sql_queries.png' | relative_url }}" alt="CockroachDB Admin UI SQL Queries graph" style="border:1px solid #eee;max-width:100%" />
+
+In the node view, the SQL Queries metric depicts the current moving average, over the last 10 seconds, of the number of SELECT/INSERT/UPDATE/DELETE queries per second issued by SQL clients on the node. 
+
+In the cluster view, the graph shows the sum of the per-node averages; that is, an aggregate estimation of the current query load over the cluster, assuming the last 10 seconds of activity per node are representative of this load.
+
+## Service Latency: SQL, 99th percentile
+<img src="{{ 'images/admin_ui_service_latency_99_percentile.png' | relative_url }}" alt="CockroachDB Admin UI Service Latency graph" style="border:1px solid #eee;max-width:100%" />
+
+Service latency is calculated as the time between receiving a query and returning a response. This time does not include network latency between the node and the client. 
+
+In the node view, the graph displays the 99th [percentile](https://en.wikipedia.org/wiki/Percentile#The_normal_distribution_and_percentiles) of service latency for the node. 
```

Comment:
Instead of linking away to an explanation of "percentile", perhaps we should work in a simple explanation, like we have in the tooltips. Essentially, 99% percentile is saying that 99% of queries were executed within this time.